src/method_ilan.jl

```diff
@@ -131,9 +133,12 @@ function ilan(
     Av=get_Av(nep)
```

Comment:
In the documentation of the solver, remember to say that the solver must be an SPMF (or at least have an implementation of `get_Av`=.